conductr_cli/sandbox_init.py

```diff
@@ -0,0 +1,51 @@
+from conductr_cli import docker_machine, terminal
+import logging
+from subprocess import CalledProcessError
+
+
+def init(args):
+    """`sandbox init` command"""
+
+    log = logging.getLogger(__name__)
+    vm_name = docker_machine.vm_name()
```

Comment:
The `docker_machine.vm_name()` does the following - is this what you have in mind?

```
DEFAULT_DOCKER_MACHINE_NAME = 'conductr'

def vm_name():
    return os.getenv('DOCKER_MACHINE_NAME', DEFAULT_DOCKER_MACHINE_NAME)
```